src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -282,6 +282,7 @@ private void startJnlpProc() throws Exception {
     @Issue("JENKINS-52165")
     @Test public void shellOutputAcrossRestart() throws Exception {
         Assume.assumeFalse("TODO not sure how to write a corresponding batch script", Functions.isWindows());
+        Assume.assumeThat("TODO no longer asserts anything, just informational. There is no way for FileMonitoringTask.Watcher to know when content has been written through to the sink other than by periodically flushing output and declining to write lastLocation until after this completes. This applies both to buffered on-master logs, and to typical cloud sinks.", System.getenv("JENKINS_URL"), nullValue());
```

Comment:
It seems like it could still be useful to assert that you haven't lost a huge amount of output, say 5% or higher.